docs/src/main/sphinx/sql/alter-table.md

```diff
@@ -54,15 +54,20 @@ per-connector basis.
 You can use the `=>` operator for passing named parameter values.
 The left side is the name of the parameter, the right side is the value being passed.
 
-Collapse files in a table that are over 128 megabytes in size with the `optimize`
-procedure from the [Hive](hive-alter-table-execute), [Delta
+Executable commands are contributed by connectors, for example, to collapse
+files in a table that are over 128 megabytes in size with the `optimize` command
+from the [Hive](hive-alter-table-execute), [Delta
 Lake](delta-lake-alter-table-execute) , and
-[Iceberg](iceberg-alter-table-execute) connectors:
+[Iceberg](iceberg-alter-table-execute) connectors, you can run the following
+query:
 
 ```
 ALTER TABLE hive.schema.test_table EXECUTE optimize(file_size_threshold => '128MB')
 ```
 
+Refer to individual [connector documentation](../connector.md) for more
```

Comment:
```suggestion
Refer to individual [connector documentation](connector) for more
```

Reply:
Also maybe this sentence is redundant anyway .. since you are already linking above